From d966d52d1e569cbc2293d841285e2b8941f28c61 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 9 Jul 2017 15:56:28 -0700
Subject: [PATCH] include linux/sysinfo.h directly

This is done to avoid the kernel header linux/kernel.h to use
__GLIBC__ define to decide on if libc implements sysinfo() API
or not. Kernel headers should be independent of such assumptions
but until its done in right place, change the local header
override to avoid this assumption

Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
Upstream-Status: Pending

 usr/include/sys/sysinfo.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/usr/include/sys/sysinfo.h
+++ b/usr/include/sys/sysinfo.h
@@ -6,7 +6,7 @@
 #define _SYS_SYSINFO_H
 
 #include <sys/types.h>
-#include <linux/kernel.h>
+#include <linux/sysinfo.h>
 
 extern int sysinfo(struct sysinfo *info);
 
